docs/docs/recipes/transforming-data.md

```diff
@@ -54,3 +54,39 @@ export const query = graphql`
 
 - [Tutorial on transforming Markdown to HTML](/tutorial/part-six/#transformer-plugins) using `gatsby-transformer-remark`
 - Browse available transformer plugins in the [Gatsby plugin library](/plugins/?=transformer)
+
+## Transforming images into grayscale using GraphQL
+
+`gatsby-image` is a React component designed to work seamlessly with Gatsby’s [native image processing](https://image-processing.gatsbyjs.org/) capabilities powered by GraphQL and [gatsby-plugin-sharp] `/packages/gatsby-plugin-sharp/` to easily and completely optimize image loading for your sites.
+
+### Prerequisites
+
+- A Gatsby site with `gatsby-config.js` and an `index.js` page
+- The `gatsby-image`, `gatsby-transformer-sharp`, and `gatsby-plugin-sharp` packages installed and added to the plugins array in `gatsby-config`
+- An image (`.jpg`, `.png`, `.gif`, `.svg`, etc.) in the `src` folder
+
+### Directions
+
+1. Set up a [GraphQL query]`/docs/graphql-reference/` and either pass it into a component as props or write it directly in the component. One technique is to leverage the <code>[useStaticQuery]`/docs/use-static-query/`</code> hook.
+
+Common GraphQL queries for sourcing images include `file` from [gatsby-source-filesystem]`/packages/gatsby-source-filesystem/`, and both `imageSharp` and `allImageSharp` from [gatsby-plugin-sharp]`/packages/gatsby-plugin-sharp/`, but ultimately the options available to you will depend on your content sources.
```

Comment:
Because we're talking about gatsby-image it would be good to include that in the GraphQL query options. Perhaps we could show an example? Something like:
```
file(relativePath: { eq: "default.jpg" }) {
  childImageSharp {
    fixed(width: 400, height: 400) {
      ...GatsbyImageSharpFixed
    }
  }
}
```

Reply:
I'd also recommend keeping in mind the recipe format, which is focused on actionable steps. Explanations and reasoning are more in the realm of guides and tutorials (and links to Additional Resources at the end can help lead users in that direction)